generators/client/templates/src/main/webapp/app/_app.config.js

```diff
@@ -1,49 +1,51 @@
-'use strict';
+(function() {
+  'use strict';
```

Comment:
@mraible since these are set on closure function for the whole file it should be fine as its similar to having at the file level IMO ofcourse we need not do for each function one closure per file would do. It will also avoid any issue when concatenating strict and non strict files